.github/workflows/ci-parallel.yml

```
@@ -0,0 +1,462 @@
+  workflow_dispatch:
+
+env:
+  NODE_VERSION: '22.18.0'
+  POSTGRES_VERSION: '15'
+
+jobs:
+  # Backend CI
+  backend-ci:
+    name: Backend CI
+    runs-on: ubuntu-latest
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)